--- free-sa-1.6.2/configs/generic-any-cc.mk.org	2009-04-16 20:32:29.343640310 +0000
+++ free-sa-1.6.2/configs/generic-any-cc.mk	2009-04-16 20:34:08.673836739 +0000
@@ -1,14 +1,14 @@
-PREFIX = /usr/local
+PREFIX = /usr
-ETCDIR = $(PREFIX)/etc/$(PROGLNAME)
+ETCDIR = /etc/$(PROGLNAME)
-WWWDIR = $(PREFIX)/var/www/htdocs/$(PROGLNAME)
-TMPDIR = $(PREFIX)/var/cache/$(PROGLNAME)
+WWWDIR = /usr/share/$(PROGLNAME)
+TMPDIR = /var/cache/$(PROGLNAME)
 LOCDIR = $(PREFIX)/share/$(PROGLNAME)
 DOCDIR = $(PREFIX)/share/doc/$(PROGLNAME)-$(PROGVERSION)
 LOGFILE = /var/log/squid/access.log
 
 CC = cc
-SORT = /usr/bin/sort
+SORT = /bin/sort
 INSTALL = $(GMKPATH)/SAinstall.sh
 
-LARGEFILE = no
+LARGEFILE = yes
 SATSEP = 
